src/app/system/global-configurations/global-configurations.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="container">
+
+  <div fxLayout="row">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+
+  <div class="mat-elevation-z8">
+
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="name">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
+        <td mat-cell *matCellDef="let configuration"> {{ configuration.name }}
+        <fa-icon icon="question-circle" matTooltip="{{ configuration.description === undefined ? 'No Description' : configuration.description }}" matTooltipPosition="right"></fa-icon>
```

Comment:
See the community-app for a suitable implementation to view all descriptions for the specific configurations. All of them currently show `No Description`.

Reply:
Are the others hardcoded in the community-app?

Reply:
By hardcoded you mean that they are showing `No Description` or not. Right?

Reply:
They are showing a description in the community-app so if it is not being returned by the API for some, then it might be hardcoded there. You will have to update here also in a similar manner that case.

Reply:
It is being fetched from API only and we are doing that only :)

Reply:
Then there is some other issue due to which some descriptions are not showing.

Reply:
They are not coming from backend as this seems to be an optional field.

Reply:
Okay, if they are not coming from the backend they must be hardcoded. So you'll have to do the same.

Reply:
It seems there some sort of confusion, so the thing is that in the API response, some configurations have description and while some does not have and in community-app when there is no description it is not showing any tooltip but does show the icon. So what I have done is that for the configurations the backend is not giving us any description, I am showing a tooltip showing  `No Description`.

Reply:
Is the community-app using the API to show the description?

Reply:
It seems that they are hard-coded in community-app but data is exactly same.

Reply:
Moreover, it seems a better approach to me.

Reply:
Similar to https://github.com/openMF/web-app/issues/236 create an issue to mention describing the problem properly and add the link to the Fineract ticket. After this, you may resolve this conversation.

Reply:
https://github.com/openMF/web-app/issues/372